setup.py

```diff
@@ -0,0 +1,12 @@
+from setuptools import setup
+
+setup(
+    name='vep-mapping-pipeline',
+    version='',
+    packages=['vep_mapping_pipeline', 'repeat_expansion_variants', 'repeat_expansion_variants.test'],
+    url='',
+    license='',
+    author='ktsukanov',
+    author_email='ktsukanov@ebi.ac.uk',
+    description=''
```

Comment:
Suggest adding a description here...

Reply:
Fixed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/4/commits/736feddd6e3df72350f37a735d668fe6f905f412